share/src/main/resources/alfresco/slingshot-application-context.xml

```diff
@@ -157,6 +157,16 @@
             <null/>
         </property>
         <property name="proxyControllerInterceptor" ref="slingshotProxyControllerInterceptor"/>
+        <property name="uriWhitelist">
+            <list>
+                <value>^.*$</value>
+            </list>
+        </property>
+        <property name="uriBlacklist">
```

Comment:
Is possible to skip this set up being already as default?

Reply:
I thought I'd leave it for informative reasons.

Reply:
Good Point. I think here a doc can help.. @DavidCervigni @skopf guys what do you think? the point is to declare or not into the prop bean the blacklist with the default value.